brainscore_language/plugins/futrell2018/benchmark.py

```diff
@@ -0,0 +1,96 @@
+import logging
+import numpy as np
+from numpy.random import RandomState
+
+from brainio.assemblies import DataAssembly
+from brainscore_core.benchmarks import BenchmarkBase
+from brainscore_core.metrics import Score, Metric
+from brainscore_language import load_dataset, load_metric, benchmarks
+from brainscore_language.artificial_subject import ArtificialSubject
+from .data import BIBTEX
+
+logger = logging.getLogger(__name__)
+
+
+class Futrell2018Pearsonr(BenchmarkBase):
+    """
+    Evaluate model ability to predict reading times on the natural stories corpus introduced in Futrell et al. 2018.
+    Alignment of reading times between model and human subjects is evaluated via Pearson correlation.
+
+    This benchmark builds off the behavioral benchmark introduced in Schrimpf et al. 2021, but:
+    * does not allow for any fitting; rather model candidates have to directly output reading times, and
+    * estimates the ceiling with Spearman-Brown corrected split-halves consistency.
+    """
+
+    def __init__(self):
+        self.data = load_dataset('Futrell2018')
+        self.metric = load_metric('pearsonr')
+        ceiler = SplitHalvesConsistency(num_splits=10, split_coordinate='subject_id', consistency_metric=self.metric)
+        super(Futrell2018Pearsonr, self).__init__(
+            identifier='Futrell2018-pearsonr',
+            version=1,
+            parent='behavior',
+            ceiling_func=lambda: ceiler(self.data),
+            bibtex=BIBTEX)
+
+    def __call__(self, candidate: ArtificialSubject) -> Score:
+        candidate.perform_behavioral_task(ArtificialSubject.Task.reading_times)
+        stimuli = self.data['word'].values
+        predictions = candidate.digest_text(stimuli)['behavior'].values
+        targets = self.data.mean('subject')  # compare to "average human"
+        raw_score = self.metric(predictions, targets)
+        score = ceiling_normalize(raw_score, self.ceiling)
+        return score
+
+
+def ceiling_normalize(raw_score, ceiling):
+    # normalize by ceiling, but not above 1
+    score = raw_score / ceiling
+    score.attrs['raw'] = raw_score
+    score.attrs['ceiling'] = ceiling
+    if score > 1:
```

Comment:
maybe this overshoot correction should go in its own separate function or allow the user to do it posthoc? it's good to have a version that has the information on if/when the ceiling was exceeded

Reply:
The overshoot value is still kept in the `score.attrs`, is that sufficient? I would like to enforce `0 < score < 1` for all benchmarks so that the model ranking doesn't get funky.
![image](https://user-images.githubusercontent.com/5308236/182663203-d18e56a1-c964-4772-8d7e-3c2c57a6ac5e.png)